src/main/containers/MockupsPanel.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useContext } from 'react';
+import styled from 'styled-components';
+import StyledPanel from './StyledPanel.jsx';
+import { TestsContext } from '../testsContext';
+import Button from '../components/Button.jsx';
+import DataTree from '../components/DataTree.jsx';
+// import NameForm from '../components/NameForm.jsx';
+import Mockup from '../components/Mockup.jsx';
+
+// will need to get data from the get request to pass to the formatted view
+const MockupsPanel = (props) => {
+  const {
+    active, datacanvas, data, onClickFunction, setTestsDiff, testsDiff,
+  } = props;
+
+  const [tests, setTests] = useContext(TestsContext);
+  const [testsListCounter, setTestsListCounter] = useState(0);
+
+  const mockupsListDisplay = [];
+  let i = 0;
+  tests.forEach((test, index) => {
+    const { name } = test;
+
+    mockupsListDisplay.push(
+      // Creates a component for each test to display name and data
+      <Mockup key={index} test={test} index={index}/>,
```

Comment:
I like this refactor a lot.  With the additional work I think it makes the files a lot more readable